coalib/results/SourceRange.py

```diff
@@ -126,6 +126,27 @@ def __json__(self, use_relpath=False):
             _dict['file'] = relpath(_dict['file'])
         return _dict
 
+    def __str__(self):
+        """
+        Default representation::
```

Comment:
add newline before and after code samples please, also can you make this a doctest?